Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -1750,3 +1753,97 @@ void MainWindow::recenterSceneView(const QModelIndex &id)
         sceneView->scrollTo(proxyModel->mapFromSource(id));
     }
 }
+
+void MainWindow::stat_dlg_update()
+{
+  if(statistics_dlg)
+    statistics_dlg->hide();
+  statistics_on_polyhedron();
+
+}
+void MainWindow::statistics_on_polyhedron()
+{
```

Comment:
Could the name of that function be independent from the item types? The name "polyhedron" should be removed.